src/lib/isTaxID.js

```diff
@@ -1226,15 +1238,26 @@ export default function isTaxID(str, locale = 'en-US') {
     if (locale in sanitizeRegexes) {
       strcopy = strcopy.replace(sanitizeRegexes[locale], '');
     }
-    if (!taxIdFormat[locale].test(strcopy)) {
-      return false;
-    }
 
-    if (locale in taxIdCheck) {
-      return taxIdCheck[locale](strcopy);
+    // Check if locale has multiple TIN
+    if (locale in multipleTinLocale) {
+      if (!localeOption) throw new Error(`Locale requires localeOption ${localeOption}`);
```

Comment:
`localeOption` is not defined so this would not work. I would suggest returning the locale

```suggestion
      if (!localeOption) throw new Error(`locale ${locale} requires localeOption`);
```

Reply:
Thought of throwing more accurate error message, any way this is also a good idea.